packages/atomic/src/components/atomic-table-result/atomic-result-cell.pcss

```diff
@@ -1,45 +1,12 @@
 @import '../../global/global.pcss';
 @import '../atomic-result/atomic-result-with-sections.pcss';
 @import '../atomic-result/atomic-result-row-desktop.pcss';
-@import '../atomic-result/atomic-result-row-mobile.pcss';
-@import '../atomic-result/atomic-result-cell-desktop.pcss';
-@import '../atomic-result/atomic-result-cell-mobile.pcss';
 
 :host {
   @apply font-sans font-normal;
 }
 
-.cell-root {
-  &.with-sections {
-    @mixin result-with-sections;
-
-    &.display-list {
-      @screen desktop-only {
-        @mixin row-result-desktop;
-      }
-      @screen mobile-only {
-        @mixin row-result-mobile;
-      }
-    }
-    &.display-grid {
-      &.image-large {
-        @screen desktop-only {
-          @mixin cell-result-desktop;
-        }
-        @screen mobile-only {
-          @mixin row-result-mobile;
-        }
-      }
-      &.image-small,
-      &.image-icon,
-      &.image-none {
-        @screen desktop-only {
-          @mixin cell-result-desktop;
-        }
-        @screen mobile-only {
-          @mixin cell-result-mobile;
-        }
-      }
-    }
-  }
+.cell-root.with-sections {
+  @mixin result-with-sections;
+  @mixin row-result-desktop;
```

Comment:
Tables aren't meant to be displayed on mobile, so I prefer to stick to the desktop look for their cells.